### DISCUSSION TITLE
.bib File Export Location Can't be Modified

### DISCUSSION THREAD

**Discussion by mmzotero** [retorquere/zotero-better-bibtex#2512] [Q&A] [answered]
Hello,

Hope you're well! I apologise if this has been asked- I couldn't find this exact issue using the search bar.

I use Zotero, Obsidian, and BBT. 

I got a new computer and I copied the entire Zotero folder onto the new computer. I downloaded BBT. But I can't change the "export to" location of BBT/.bib files on my new computer under the Preferences -> Automatic Export tab. It is still stuck on the file name from my old computer which is the wrong file name. (If I click the "down arrow" button, it won't let me browse for another folder to save into or type anything else.)

Consequently, I can't sync my Citations into Obsidian.

Does anyone have any ideas? Thank you so much!!

<img width="495" alt="image" src="https://github.com/retorquere/zotero-better-bibtex/assets/133684068/dcb65184-f2ab-448c-ba49-91656972f7c2">

**Comment by retorquere** [comment]
Only way is to remove it and set up a new one.

**Reply by mmzotero** [reply]
Thank you!

Sorry, like start my Zotero from scratch? (Not just re-load the old saved up backup I have?)

**Reply by retorquere** [reply, marked as answer]
No, at the bottom of the screen that you made a screenshot from there is a button "remove". That just removes this auto export.

**Reply by mmzotero** [reply]
Thank you! I did that! But now when I try to Export Library it says Error Occurred.. Do I start an "issues" thing? I already reset the cache like your other thread said.

**Reply by retorquere** [reply]
Yes, please open a new issue.
